### DISCUSSION TITLE
Should this be run without PerfectWeave?

### DISCUSSION THREAD

**Discussion by mckenna654** [256shadesofgrey/eso-light-attack-weave#22] [General]
Hi,

Just wondering if anyones tried running this AHK script with PerfectWeave to see if it parses better?

PerfectWeave essentially blocks you from casting an ability too soon without light attacking, to avoid missing the weave. https://www.esoui.com/downloads/info2918-PerfectWeave.html

Just wondering how running this AHK script would function with perfect weave?

**Comment by 256shadesofgrey** [comment]
I never used the addon you linked, but from what I understand based on the description, it should theoretically not interfere, because as long as you have queuing enabled in the script (it's on by default), it also waits for the GCD to end. But because I don't know what exactly the addon bases its lockout on (whether it's a simple timer, or if it can track directly if skills can be used), AutoHotkey has a not very precise clock, and because latency might have an effect, there might be race conditions and the addon might occasionally prevent a skill from firing, and then you might end up with 2 LAs before the skill.

All I can really say is, just try it, and see how it goes.
